infrastructure/templates/search/ci/jobs/deploy-infrastructure.yml

```diff
@@ -67,7 +67,6 @@ jobs:
                 serviceConnection: ${{ parameters.serviceConnection }}
                 indexCorsAllowedOrigins:
                   - $(publicSiteUrl)
-                  - 'https://localhost:3000'
```

Comment:
I think we should make a note to remember to test that this has been updated after the next Search Infrastructure deploy, starting with Dev.